packages/near-fast-auth-signer/src/components/SignMultichain/utils.ts

```diff
@@ -10,14 +9,43 @@ import {
   MultichainInterface
 } from './types';
 import { assertNever } from '../../utils';
-import { Bitcoin } from '../../utils/multi-chain/chains/Bitcoin';
-import signAndSend, { getEstimatedFeeBTC, getEstimatedFeeEVM } from '../../utils/multi-chain/multiChain';
+import { networkId } from '../../utils/config';
+import { Bitcoin } from '../../utils/multi-chain/chains/Bitcoin/Bitcoin';
+import {
+  fetchBTCFeeProperties,
+  signAndSendBTCTransaction,
+  signAndSendEVMTransaction
+} from '../../utils/multi-chain/multiChain';
+import { fetchEVMFeeProperties } from '../../utils/multi-chain/utils';
 import { fetchGeckoPrices } from '../Sign/Values/fiatValueManager';
 
 // TODO: use this for blacklisting on limited access key creation AND sign
 const MULTICHAIN_CONTRACT_TESTNET = 'multichain-testnet-2.testnet';
 const MULTICHAIN_CONTRACT_MAINNET = 'multichain-testnet-2.testnet';
 
+type BTCFeeProperites = {
+  inputs: {
```

Comment:
Re-use UTXO type here

Reply:
Don't think we should export anything from the module aside from the multi chain file for now to prevent unpredictable usage paths